plugins/ibm_qradar/help.md

```diff
@@ -0,0 +1,1068 @@
+
+Example input:
+
+```
+{
+  "assigned_to": "admin",
+  "closing_reason_id": 100,
+  "fields": "id",
+  "follow_up": false,
+  "offense_id": 100,
+  "protected": false,
+  "status": "OPEN"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|data|offense|False|JSON Data of the Offense|
+
+Example output:
+
+```
+{
+  "data": {
+    "event_count": 4,
+    "last_updated_time": 1640155212603,
+    "magnitude": 3,
+    "start_time": 1640155212500,
+    "description": "Multiple Exploit/Malware Types Targeting a Single ...",
+    "first_persisted_time": 1640155213000,
+    "follow_up": false,
+    "inactive": true,
+    "relevance": 0,
+    "source_count": 1,
+    "device_count": 2,
+    "rules": [
+      {
+        "id": 100113,
+        "type": "CRE_RULE"
+      }
+    ],
+    "security_category_count": 3,
+    "policy_category_count": 0,
+    "source_network": "other",
+    "credibility": 3,
+    "flow_count": 0,
+    "local_destination_count": 0,
+    "offense_type": 1,
+    "remote_destination_count": 1,
+    "source_address_ids": [
+      18
+    ],
+    "categories": [
+      "Command Execution",
+      "Input Validation Exploit"
+    ],
+    "id": 33,
+    "offense_source": "49.206.13.49",
+    "protected": false,
+    "domain_id": 0,
+    "last_persisted_time": 1642054784000,
+    "category_count": 3,
+    "destination_networks": [
+      "other"
+    ],
+    "local_destination_address_ids": [],
+    "log_sources": [
+      {
+        "type_name": "VirsecSecurityPlatformCustom",
+        "id": 23312,
+        "name": "Virsec Security Platform",
+        "type_id": 4013
+      },
+      {
+        "type_id": 18,
+        "type_name": "EventCRE",
+        "id": 63,
+        "name": "Custom Rule Engine-8 :: ip-172-31-34-93"
+      }
+    ],
+    "severity": 9,
+    "status": "OPEN",
+    "username_count": 0
+  }
+}
+```
+
+### Triggers
+
+#### Get New Offenses
+
+This trigger is used to list all New Offenses.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|fields|string|None|False|Use this parameter to specify which fields you would like to get back in the response. Fields that are not named are excluded. Specify subfields in brackets and multiple fields in the same object are separated by commas|None|id|
+|filter|string|None|False|This parameter is used to restrict the elements in a list base on the contents of various fields|None|id=100|
+|interval|integer|15|True|How frequently (in seconds) to trigger a greeting|None|15|
+|range|string|None|False|Use this parameter to restrict the number of elements that are returned in the list to a specified range. The list is indexed starting at zero|None|1-2|
+|sort|string|None|False|Use parameter to sort the elements in a list|None|+id|
```

Comment:
Same as below. Do we need + before the field name for sorting

Reply:
Yes, we need + or - sign before the field name. As it indicates the order or sorting + indicates the ascending order and - indicates the descending sort.